src/System.Private.Uri/src/System/UriExt.cs

```diff
@@ -499,29 +499,46 @@ internal unsafe bool InternalIsWellFormedOriginalString()
         public static string UnescapeDataString(string stringToUnescape)
         {
             if ((object)stringToUnescape == null)
+            {
```

Comment:
my slight preference is to omit them here as it's easier to read but either is fine

---
In reply to: [103753759](https://github.com/dotnet/corefx/pull/16456#discussion_r103753759) [](ancestors = 103753759)